src/plugins/mf_priority.cpp

```diff
@@ -295,12 +372,29 @@ static int inactive_cb (flux_plugin_t *p,
                                                     FLUX_JOBTAP_CURRENT_JOB,
                                                     "mf_priority:bank_info"));
 
-    if (b == NULL)
+    if (b == NULL) {
         flux_jobtap_raise_exception (p, FLUX_JOBTAP_CURRENT_JOB, "plugin",
-                                     3, "mf_priority: bank info is missing");
```

Comment:
Sorry I didn't notice this before, but lets use fatal exception severity for now.